envGenerate/template/Makefile

```
@@ -0,0 +1,101 @@
+JSONNET_VERSION := 0.18.0
+YQ_VERSION := 4.24.5
+AUTOGEN_COMMENT := This file is automatically generated from a template. Please do not edit.
+
+ROOT_DIR := $(shell git rev-parse --show-toplevel)
+BINDIR := $(abspath $(CURDIR)/bin)
+JSONNET := $(BINDIR)/jsonnet
+JSONNETFMT := $(BINDIR)/jsonnetfmt
+YQ := $(BINDIR)/yq
+
+DOWNLOAD_DIR := $(ROOT_DIR)/download
+JSONNET_DLPATH := $(DOWNLOAD_DIR)/jsonnet-v$(JSONNET_VERSION).tar.gz
+YQ_DLPATH := $(DOWNLOAD_DIR)/yq-v$(YQ_VERSION).tar.gz
+
+TEMP_DIR := /tmp/site-env
+
+WGET := wget --retry-connrefused --no-verbose
+WGET_PATH := $(shell which wget || echo wget-install)
+
+.PHONY: all
+all: site-env common-env
+
+write-%:
+	mkdir -p $(TEMP_DIR)
+	$(JSONNET) $*.jsonnet > $(TEMP_DIR)/$*.json
+	set -e; for i in $$(cat $(TEMP_DIR)/$*.json | jq -r 'keys | .[]'); do \
+		echo $$i; \
+		mkdir -p $(ROOT_DIR)/$$(dirname $$i); \
+		cat $(TEMP_DIR)/$*.json |jq 'del(.[][] | select(length == 0))' | $(YQ) --no-doc ".[\"$$i\"] | .[0] headComment=\"$(AUTOGEN_COMMENT)\" | .[] | splitDoc" - --prettyPrint > $(ROOT_DIR)/$$i; \
```

`--no-doc` seems to cancel `| splitDoc`.

```suggestion
		cat $(TEMP_DIR)/$*.json |jq 'del(.[][] | select(length == 0))' | $(YQ) ".[\"$$i\"] | .[0] headComment=\"$(AUTOGEN_COMMENT)\" | .[]" - --prettyPrint > $(ROOT_DIR)/$$i; \
```

---

Removed `--no-doc`, `| splitDoc` and configured the array not to split into multiple documents.
 08840932a8458590a06d7695a5c2bceb0e5a338d.